### ISSUE TITLE
How do we want to deal with ancient, no activity issues and pull requests?

### ISSUE THREAD

**Issue by dwijnand** [sbt/sbt#2390]
We have a bunch of ancient issues and pull requests where no one's ever come back, or generally hasn't had any activity. I don't mean legitimate issues that were opened a long time ago, I mostly mean NeedsInfo bug reports and abandoned pull requests.

Do we want to find a system or a tool to automate away this bloat?

**Comment by eed3si9n** [comment]
In general sbt has asymmetry of user and contributor ratio, so as the developers, the best we can do is organize the development process so it's easy to contribute to sbt development. As Linus says "given enough eyeballs, all bugs are shallow."

**Comment by dwijnand** [comment]
I agree to that, but what do we want to do about this problem? Or are you saying it's not a problem and leave it as it is?

**Comment by eed3si9n** [comment]
More or less. I used to try to close as many issues as possible, but I feel like it creates unnecessary friction, and our time would probably be better spent encouraging people to do more right things, rather trying to discourage people of doing the wrong things.

One of the things I would like to do more is answer sbt questions more regularly on Stackoverflow, and advertise that I answered it, so people would know that's where they can get my attention.
